docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md

```diff
@@ -135,8 +119,8 @@ Assign the Rancher AKSv2 role to the service principal with the Azure Command Li
 
 ```
 az role assignment create \
---assignee CLIENT_ID \
---scope "/subscriptions/SUBSCRIPTION_ID/resourceGroups/RESOURCE_GROUP_NAME" \
+--assignee <client-id> \
+--scope "/subscriptions/<subscription-id>/resourceGroups/<resource-group-name>" \
```

Comment:
```suggestion
--scope "/subscriptions/<subscription-id>/resourceGroups/<resource-group>" \
```

Lines 36, 54, and 123 use `<group>`, `<azure-resource-group-name>`, and `<resource-group-name>` as placeholders for resource group. These should match since.